dotnet/src/Connectors/Connectors.AI.LLama/Directory.Packages.props

```diff
@@ -0,0 +1,7 @@
+
+<Project>
+  <ItemGroup>
+    <PackageReference Include="LLamaSharp" Version="0.4.0" />
```

Comment:
Please move these to the common Directory.Packages.props under /dotnet

Reply:
moved ok